go.mod

```diff
@@ -35,5 +35,7 @@ require (
 	github.com/syndtr/gocapability v0.0.0-20200815063812-42c35b437635
 	golang.org/x/sync v0.0.0-20210220032951-036812b2e83c
 	golang.org/x/sys v0.0.0-20210820121016-41cdb8703e55
-	golang.org/x/term v0.0.0-20201126162022-7de9c90e9dd1
+	golang.org/x/term v0.0.0-20210615171337-6886f2dfbf5b
 )
+
+replace github.com/containers/image/v5 => ../image
```

Comment:
Note that you could replace it with your fork on GitHub as well so that CI could fire.